.github/workflows/tests.yml

```diff
@@ -39,6 +39,7 @@ jobs:
           && !contains(github.event.pull_request.labels.*.name, 'ci:run-full-test-suite')
           || false
          }}
+      pytest_store_durations: false

```

Comment:
Set this to `true` to upload durations as artifacts.

Reply:
With the argument set to true, the longest test took 17:28 mins (see [here](https://github.com/PennyLaneAI/pennylane/actions/runs/10710884918?pr=6214)), and with the argument set to false the longest test took 17:17 mins (see [here](https://github.com/PennyLaneAI/pennylane/actions/runs/10710684906/job/29697940552?pr=6214)).

This is is a negligible difference, so clearly setting the `--store-durations` argument for `pytest` doesn't add any meaningful overhead to the workflow. We could in theory just _always_ store durations and upload artifacts.